python/benchmark/BENCHMARK.md

```
@@ -1,267 +1,353 @@
+  Reqs/sec    243276.66   32740.99  312561.12
+  Latency      404.91us   380.86us    11.70ms
   Latency Distribution
-     50%   266.00us
-     75%   375.00us
-     90%   552.00us
-     99%     1.97ms
+     50%   268.00us
+     75%   413.00us
+     90%   735.00us
+     99%     2.89ms
 ### Redirect Response (/redirect)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)